src/scss/tom-select.scss

```diff
@@ -159,7 +150,6 @@ $select-spinner-border-color:					$select-color-border !default;
 	font-family: $select-font-family;
 	font-size: $select-font-size;
 	line-height: $select-line-height;
-	font-smoothing: $select-font-smoothing;
```

Comment:
Another unrecognised property, and its accompanying SCSS var, removed, might want putting back in?

Reply:
When the build CSS is compiled this is not prefixed with `--webkit-`